pkg/reconciler/instances/istio/bootstrap_test.go

```diff
@@ -113,9 +115,68 @@ func TestParsePaths(t *testing.T) {
 		}
 
 		//when
-		_, err := parsePaths(strings.Join(paths, ":"), alwaysValidFn)
+		_, err := parsePaths(strings.Join(paths, ";"), alwaysValidFn, &zap.SugaredLogger{})
 		//then
 		require.Error(t, err)
 		require.Contains(t, err.Error(), "ISTIOCTL_PATH env variable exceeds the maximum istio path limit")
 	})
 }
+
+func TestChmodExecutbale(t *testing.T) {
+	t.Run("chmod to 0777", func(t *testing.T) {
+		pathToFile := "tmp/dat1"
+		t.Cleanup(func() {
+			err := os.RemoveAll("tmp")
+			require.NoError(t, err)
+		})
+		require.NoError(t, os.MkdirAll("tmp", 0777))
+		d1 := []byte("hello\nworld\n")
+		err := os.WriteFile(pathToFile, d1, 0000)
+		require.NoError(t, err)
+		err = chmodExecutbale(pathToFile, zap.NewNop().Sugar())
+		require.NoError(t, err)
+		stat, err := os.Stat(pathToFile)
+		require.NoError(t, err)
+		require.Equal(t, os.FileMode(0777), stat.Mode())
+	})
+	t.Run("chmodExecutable should return an error if file is not existing", func(t *testing.T) {
+		pathToFile := "not-existing"
+		err := chmodExecutbale(pathToFile, zap.NewNop().Sugar())
+		require.Error(t, err)
+	})
+	t.Run("chmodExecutable should return an error if pathToFile is empty", func(t *testing.T) {
+		pathToFile := ""
+		err := chmodExecutbale(pathToFile, zap.NewNop().Sugar())
+		require.Error(t, err)
+	})
+}
+
+func TestValidatePath(t *testing.T) {
+	t.Run("validatePath should return nil and change mode of file to 0777", func(t *testing.T) {
```

Comment:
You don't need to repeat the name of the tested function as this one is already in the outer testing function